spec/index.bs

```diff
@@ -2759,6 +2774,38 @@ dictionary GPUFenceDescriptor : GPUObjectDescriptorBase {
 </script>
 
 
+Queries {#queries}
+================
+
+## QuerySet ## {#queryset}
+
+<script type=idl>
+interface GPUQuerySet {
+    void destroy();
+};
+GPUQuerySet includes GPUObjectBase;
+</script>
+
+### Creation ### {#queryset-creation}
+
+<script type=idl>
+dictionary GPUQuerySetDescriptor : GPUObjectDescriptorBase {
+    required GPUQueryType type;
+    required GPUSize32 count;
+};
+</script>
+
+## QueryType ## {#querytype}
+
+<script type=idl>
+enum GPUQueryType {
+    "occlusion",
+    "pipeline-statistics",
```

Comment:
nit: only occlusion should be added for now because the others will be in other PRs

Reply:
+1 from myles and myself

Reply:
+1